web/src/stream_list_sort.ts

```diff
@@ -23,11 +23,8 @@ let all_streams: number[] = [];
 // to avoid making left sidebar rendering a quadratic operation.
 let filter_out_inactives = false;
 
-export function get_streams(): string[] {
-    return all_streams.flatMap((stream_id) => {
-        const stream_name = sub_store.maybe_get_stream_name(stream_id);
-        return stream_name === undefined ? [] : [stream_name];
-    });
+export function get_stream_ids(): number[] {
+    return all_streams;
```

Comment:
Should this do a shallow copy? Seems potentially safer.

Reply:
Yeah that makes sense, I'll do that